src/irmin-pack/content_addressable.ml

```diff
@@ -24,307 +24,11 @@ module Table (K : Irmin.Hash.S) = Hashtbl.Make (struct
   let equal = Irmin.Type.(unstage (equal K.t))
 end)
```

Comment:
you can also remove the `Table` module, as it is duplicated in pack_store.ml now